lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -730,3 +730,21 @@ function dosomething_global_user_details($account = NULL) {
 
   return array($language, $country_code);
 }
+
+/**
+ *  Lookup global related details about a campaign:
+ *  - language
+ *  - country_code
+ *
+ *  @return array
+ *    Details about the user account:
+ *      - language
+ *      - country_code
+ */
+function dosomething_global_campaign_details($node) {
+
+  $language = $node->language;
```

Comment:
If you want to grab the country code from the URL 

dosomething_global_get_current_prefix() 

However I suggest using dosomething_global_get_language() instead for this